src/main/scala/cromwell/engine/backend/jes/JesBackendCall.scala

```diff
@@ -63,5 +65,25 @@ class JesBackendCall(val backend: JesBackend,
    */
   def globOutputPath(glob: String) = s"$callGcsPath/glob-${glob.md5Sum}/"
 
-  override def resume(jobKey: JobKey) = backend.resume(this, jobKey)
+  /** TODO creating the JES run should be async but currently is not. */
+  override def execute = Future.successful(backend.execute(this))
```

Comment:
This isn't a "review" per se, more a question. I've never seen this before-- why not run `backend.execute(this)` asynchronously? Thread safety, throttling, etc?

I've only ever seen variables or constants passed to `Future.successful`, not a method-to-be-run. In other words, this current `override def execute` is synchronously blocking like:

``` scala
override def execute: Future[ExecutionHandle] = {
  val handleValue = backend.execute(this) // Block, and synchronously wait for the result
  Future.successful(handleValue) // Later (minutes, hours?), return our wrapped handle
}
```

Versus letting `Future` run asynchronously for you:
``` scala
override def execute: Future[ExecutionHandle] = {
  def handleFunction = backend.execute(this) // This is some function that will run
  Future.apply(handleFunction) // Return right now, and later complete the future
}
```